### ISSUE TITLE
[Question]: Unable to Capture complete console logs from browser using playwright

### ISSUE THREAD

**Issue by chetanSrajiv** [microsoft/playwright#7561]
### Your question

When I am using below snippet, I see that the code is printing only one line of log
```
    const browser = await chromium.launch({ headless: false, slowMo: 100 });
    const context = await browser.newContext();
    const page = await context.newPage();

    await page.goto('https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_console_error');
    await page.click("//button[text()='Run »']");

    const errorLogs = [];
    page.on("console", (message) => {
        if (message.type() === "error") {
            errorLogs.push(message.text())
        }
    })
    await page.evaluate(() => {
        console.error("hello from the browser")
    })
    console.log(errorLogs);
    await page.close();
    await context.close();
    await browser.close();
```

Output is : [ 'hello from the browser' ]

other details: playwright : "version": "1.12.3",
![w3SchoolsConsoleLogsPage](https://user-images.githubusercontent.com/13467788/125290620-24371800-e33e-11eb-9a23-b4961d356b95.png)

**Comment by pavelfeldman** [comment]
Until Playwright 1.13 (next version), we did not consider page errors as console messages. You only have single console.error call in your code, so you had one line in output.

Starting with 1.13 we included page errors in console messages. Before it releases, you can listen to 'pageerror' event to get those as a workaround.

**Comment by pavelfeldman** [comment]
I'll close this as fixed since it is addressed in v1.13.
